SWLOR.Game.Server/Perk/ForceControl/ForceHeal.cs

```diff
@@ -37,6 +37,7 @@ public float CooldownTime(NWCreature oPC, float baseCooldownTime, int spellTier)
 
         public void OnImpact(NWCreature creature, NWObject target, int perkLevel, int spellTier)
         {
+            creature.AssignCommand(() => _.ActionPlayAnimation(Animation.LoopingMeditate, 1.0f, 9999));
```

Comment:
This animation is going to stop playing as soon as the player moves. Is that ok?

Reply:
Yeah its fine, if it becomes an issue I will look for a better option.